lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -503,17 +503,29 @@ enum {
 } // namespace TypeMember
 
 namespace TypeInheritance {
+namespace NonSemantic {
+enum {
+  ParentIdx       = 0,
+  OffsetIdx       = 1,
+  SizeIdx         = 2,
+  FlagsIdx        = 3,
+  MinOperandCount = 3,
```

Comment:
`MinOperandCount` is not needed here and in OpenCL namespace as long as we have separated them. There are no optional operands in `TypeInheritance`

Reply:
Addressed